<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialogClass</class>
 <widget class="QDialog" name="NewProjectDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewProjectDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelProjectName">
     <property name="text">
      <string>Project Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditProjectName"/>
   </item>
   <item>
    <widget class="QCheckBox" name="boxEmptyProject">
     <property name="text">
      <string>Create Empty Project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxProjectFolder">
     <property name="title">
      <string>Project Folder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditProjectFolder"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonProjectFolder">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxInputFile">
     <property name="title">
      <string>Input File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="layoutFileType" stretch="0,0,0">
        <item>
         <widget class="QLabel" name="labelFileType">
          <property name="text">
           <string>File Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="boxFileType"/>
        </item>
        <item>
         <spacer name="spacerFileType">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutInputFile">
        <item>
         <widget class="QLineEdit" name="lineEditInputFile"/>
        </item>
        <item>
         <widget class="QPushButton" name="buttonInputFileName">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacerDialog">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonsDialog">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>boxEmptyProject</sender>
   <signal>toggled(bool)</signal>
   <receiver>boxInputFile</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonsDialog</sender>
   <signal>accepted()</signal>
   <receiver>NewProjectDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonsDialog</sender>
   <signal>rejected()</signal>
   <receiver>NewProjectDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
